Extend GCP broker with external database plan (#113)
kyma-project/addons: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/overview.md

The Google Cloud Platform Service Broker class exposes services from the [Google Cloud Platform Service Broker](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker)
in a given Namespace of the Kyma cluster.

> The MySQL database is used to keep the broker's state. You can configure external database. Uninstalling the broker will delete all information about created instances but created services will still exist.

Reviewer: ```suggestion
The MySQL database is used to keep the broker's state. You can configure an external database. Uninstalling the broker will delete all information about created instances but created services will still exist.
```

> The MySQL database is used to keep the broker's state. You can configure external database. Uninstalling the broker will delete all information about created instances but created services will still exist.

Reviewer: ```suggestion
The MySQL database is used to keep the broker's state. You can configure an external database. Uninstalling the broker deletes all information about created instances. Uninstalling the broker does not delete the services you created.
```

7. Create a Secret from the JSON file by running this command:

> ```

Reviewer 2: revert that, using ` it will not work properly on UI
